addon/components/cf-field/input/powerselect.js

```diff
@@ -1,57 +1,50 @@
 import { getOwner } from "@ember/application";
-import Component from "@ember/component";
-import { computed } from "@ember/object";
+import { action } from "@ember/object";
 import { inject as service } from "@ember/service";
-import { queryManager } from "ember-apollo-client";
+import Component from "@glimmer/component";
 
 /**
  * Dropdown component for the single and multiple choice question type
  *
  * @class CfFieldInputPowerSelectComponent
  * @argument {Field} field The field for this input type
  */
-export default Component.extend({
-  tagName: "",
+export default class CfContentInputPowerselectComponent extends Component {
```

Comment:
Why not `CfFieldInputPowerselectComponent`? Maybe I'm missing something though.

Reply:
Fixed